examples/terraform/druid-migration/variables.tf

```diff
@@ -0,0 +1,316 @@
+  default     = "us-central1"
+}
+
+variable zone {
+  description = "Zone in the region for the resources to be created."
+  default     = "us-central1-a"
+}
+
+variable bucket_name {
+  description = "Name for the bucket to be created, it will be used for storing the initialization script, dataproc logs and hadoop configuration files."
+  default     = "<bucket_name>"
+}
+
+variable dataproc_cluster_name {
+  description = "Cluster name for dataproc."
+  default     = "<dataproc_cluster_name>"
+}
+
+variable cloud_sql_instance_name {
+  description = "Name of the CloudSQL resource to be created."
+  default     = "<sql_instance_name>"
+}
+
+variable historical_node_count {
+  description = "Number of Historical node to be created."
+  default     = 2
+}
+
+variable middlemanager_node_count {
+  description = "Number of Middlemanager node to be created."
+  default     = 1
+}
+
+variable coordinator_node_count {
+  description = "Number of Cordinator node of druid to be created."
+  default     = 1
+}
+
+variable broker_node_count {
+  description = "Number of Broker node of druid to be created."
+  default     = 2
+}
+
+variable overlord_node_count {
+  description = "Number of Overlord node of druid to be created."
+  default     = 1
+}
+
+variable router_node_count {
+  description = "Number of Overlord node of druid to be created."
+  default     = 1
+}
+
+variable historical_node_name {
+  description = "Name of the Historical node"
+  default     = "<historical_name>"
+}
+
+variable router_node_name {
+  description = "Name of the Historical node"
+  default     = "<router_node_name>"
+}
+
+variable middlemanager_node_name {
+  description = "Name of the Middlemanager node"
+  default     = "<middlemanager_node_name>"
+}
+
+variable broker_node_name {
+  description = "Name of the Broker node"
+  default     = "<broker_node_name>"
+}
+
+variable coordinator_node_name {
+  description = "Name of the Coordinator node"
+  default     = "<coordinator_node_name>"
+}
+
+variable overlord_node_name {
+  description = "Name of the Overlord node"
+  default     = "<overlord_node_name>"
+}
+
+variable zookeeper_name {
+  description = "Name for zookeeper nodes"
+  default     = "<zookeeper_node>"
+}
+
+variable zookeeper_machine_type {
+  description = "Zookeeper cluster machine type."
+  default     = "n1-standard-1"
+}
+
+variable kafka_machine_type {
+  description = "Kafka cluster machine type."
+  default     = "n1-standard-1"
+}
+
+variable zookeeper_disk {
+  description = "Zookeeper Operating System."
+  default     = "ubuntu-os-cloud/ubuntu-1604-lts"
+}
+
+variable kafka_name {
+  description = "Name for kafka nodes"
+  default     = "<kafka_node_name>"
+}
+
+variable kafka_disk {
+  description = "Kafka Operating System."
+  default     = "ubuntu-os-cloud/ubuntu-1604-lts"
+}
+
+variable zookeeper_ip_list {
+  description = "This variable is used internally for storing the list of Zookeeper clusters IP Address"
```

Comment:
If a variable is only used internally, why are we exposing it? Variables should only be used for items which users are expected or able to override/edit.